embark/.gitignore

```diff
@@ -18,3 +18,5 @@ erl_crash.dump
 
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
+
+hello_tokumei/
```

Comment:
This shouldn't be added. You wouldn't normally be running the generator within this project

Reply:
The instructions in the Radme say to run the docker command inside `embark` dir and then we get the `hello_tokumei` folder that can be accidentally committed.

Reply:
which README? In the embark/README.md The instructions for contributing do not include running the container.
The line about beginning a new project using `docker run -v $(pwd):/tmp tokumei/embark hello_tokumei` That can be done anywhere. And typical use would be that it's not run in this project because a consumer of this project would not have the embark project on their machine

Reply:
also this command should really read 
```
docker run -v $(pwd):/tmp tokumei/embark <app_name>
```
i.e. you would provide any name you wanted your started project to have